worker/build.rs

```diff
@@ -110,62 +104,49 @@ fn main() {
         println!("cargo:rustc-link-lib=dylib=c++");
         println!("cargo:rustc-link-lib=dylib=c++abi");
     }
-    #[cfg(target_os = "windows")]
-    {
-        if !std::path::Path::new("worker/out/msys/bin/make.exe").exists() {
-            let python = if let Ok(python) = env::var("PYTHON") {
-                python
-            } else if Command::new("where")
-                .arg("python3")
-                .status()
-                .expect("Failed to start")
-                .success()
-            {
-                "python3".to_string()
-            } else {
-                "python".to_string()
-            };
-
-            let dir = format!("{}/msys", mediasoup_out_dir.replace('\\', "/"));
-
-            if !Command::new(python)
-                .arg("scripts\\getmake.py")
-                .arg("--dir")
-                .arg(dir.clone())
-                .status()
-                .expect("Failed to start")
-                .success()
-            {
-                panic!("Failed to install MSYS/make")
-            }
-
-            env::set_var(
-                "PATH",
-                format!("{}\\bin;{}", dir, env::var("PATH").unwrap()),
-            );
-        }
 
-        env::set_var(
-            "PATH",
-            format!(
-                "{}\\worker\\out\\msys\\bin;{}",
-                env::current_dir()
-                    .unwrap()
-                    .into_os_string()
-                    .into_string()
-                    .unwrap(),
-                env::var("PATH").unwrap()
-            ),
-        );
+    // Install Python invoke package in custom folder
+    let pip_invoke_dir = format!("{out_dir}/pip_invoke");
+    let invoke_version = "2.2.0";
```

Comment:
Is there a good reason why we need this specific version? By not specifying a version we'll get support for future Python versions and bugfixes for free.

If we do need it, this should be a constant at the top of the file instead.

Reply:
I don't know yet how Invoke library behaves regarding compatibility or breaking changes. I think it's ok to force version 2.2.0 which we know it works in all archs. This doesn't mean we should never upgrade it. But I'm not confident enough to say ·any future version will work without changes anywhere".